integr8ly/integreatly-operator
# MGDAPI-3315 Add CRO release v0.35.2 to RHOAM

# Issue link

MGDAPI-3315

# What

Add CRO release v0.35.2 to RHOAM
# Verification steps
- Install RHOAM on a byoc cluster with useClusterStorage=false.
```bash
INSTALLATION_TYPE=managed-api make cluster/prepare/local
INSTALLATION_TYPE=managed-api USE_CLUSTER_STORAGE=true make deploy/integreatly-rhmi-cr.yml
INSTALLATION_TYPE=managed-api make code/run 
```
- Confirm that a new version of CRO is present
- Confirm that Postgres, Redis and blobstorage resources are provisioned correctly.
